### DISCUSSION TITLE
Flash Builder 4.7 flex compiler arguments for iOS

### DISCUSSION THREAD

**Discussion by arminkapp** [airsdk/Adobe-Runtime-Support#1403] [Q&A]
Hi!
I accidentally deleted the additional flex complier arguments needed to target iOS in flash builder 4.7  
Would anyone please be able to help me out and let me know which arguments need to be included? Or where can I find documentation regarding complier arguments?
I am using apache flex 4.16.1 with harman air 33.1.663
These are the arguments I have listed:
-locale en_US  -swf-version=44

Whichever argument I accidentally deleted, causes the app to ignore all the ios specific skins when I run the simulator, and it won't compile at all as an ipa release build.

Many thanks!!!

**Comment by 2jfw** [comment]
In FlashBuilder, you could check the specific project file, where the changes where made, and check the local history to see if there is a previous version available.

**Reply by arminkapp** [reply]
Thank you for the tip.  I checked after receiving your message and unfortunately, there is no previous version available in the local history folder.

**Comment by 2jfw** [comment]
As far as I know, there are no specific invoke arguments required in order to run/build an iOS app.
To me, this rather sounds like there is a switch in your applications source code where, depending on the injected invoke arguments, a specific theme is used.
Try to look for some function or logic related to the `flash.events.InvokeEvent`
